__tests__/register-e2e-test.js

```diff
@@ -12,15 +12,17 @@ it('e2e', () => {
   )
 
   // claim a wrapper
-  experience({ owner: 'owner123' }).register(['wrapper'], (slots, React) => {
+  experience({ owner: 'owner123' }).register(['wrapper'], async (slots, React) => {
     expect(mounted.find('.wrapped').length).toEqual(1)
     expect(mounted.find('.replaced').length).toEqual(0)
 
     slots.render('wrapper', () => { return <div className='replaced' /> })
+    await defer()
```

Comment:
This is because React 16 has async rendering with the arrival of fiber.

Reply:
wait, async rendering is on by default on 16?

Reply:
Not sure, I think so. Its the only way i got these tests to pass.